data_steward/utils/participant_summary_requests.py

```diff
@@ -101,7 +120,64 @@ def get_deactivated_participants(project_id, columns):
                     item.append(val)
         deactivated_participants.append(item)
 
-    df = pandas.DataFrame(deactivated_participants,
+    # Transforms participantId to an integer string
+    updated_deactivated_participants = participant_id_to_int(deactivated_participants, columns)
+
+    df = pandas.DataFrame(updated_deactivated_participants,
                           columns=deactivated_participants_cols)
 
-    return df
+    # To store dataframe in a BQ dataset table
+    destination_table = dataset_id + '.' + tablename
+
+    dataset = store_participant_data(df, project_id, destination_table)
+
+    return dataset
```

Comment:
Based on how you decide to / not to return values from `store_participant_data`, the `dataset` variable may be `None`.  If it is `None`, you **may** want to do something like:
```suggestion
    store_participant_data(df, project_id, destination_table)

    return '.'.join([project_id, destination_table])
```

Reply:
Yep, `dataset` was returning `None`. I took your suggestion and changed the return statement of `get_deactivated_participants`